src/services/tracing.ts

```diff
@@ -171,6 +171,78 @@ export const asyncRunInSpan = <F extends () => ReturnType<F>>(
   return ret
 }
 
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
+export const wrapToRunInSpan = <A extends Array<any>, R>({
```

Comment:
`unknown` doesn't work instead of `any`?

Reply:
is giving me type errors :thinking: do you have some suggestion to implement it? cc @samerbuna